users/src/main/java/com/maksimpegov/users/user/UserDto.java

```diff
@@ -0,0 +1,16 @@
+package com.maksimpegov.users.user;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class UserDto {
+    private String username;
+    private String password;
```

Comment:
Does that mean you returning to the client username and password? You newer should return password from API.